compiler/aarch64/codegen/OMRRegisterDependency.cpp

```diff
@@ -419,6 +419,18 @@ void OMR::ARM64::RegisterDependencyConditions::stopUsingDepRegs(TR::CodeGenerato
       _postConditions->stopUsingDepRegs(getAddCursorForPost(), numRetReg, retReg, cg);
    }
 
+
+/**
+ * @brief Answers if the register dependency is KillVectorRegs
+ *
+ * @param[in] dep : register dependency
+ * @return true if the register dependency is KillVectorRegs
+ */
+static bool isKillVectorRegs(TR::RegisterDependency *dep)
```

Comment:
I think `killsVectorRegs()` would be a better name for this function.

Reply:
Changed the function name to `killVectorRegs`.

Reply:
Changed the function name again to `killsVectorRegs`.